# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1292653254 28800
#      Fri Dec 17 22:20:54 2010 -0800
# Node ID 027fbc3898982bb9fac18d52f88c1c8f8cbe8423
# Parent  2750b04a1b5c9bdf27cb23b9fe50dbe9dc022984
Remove seealso reference to deprecated function dispatch.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-12-17  Rik  <octave@nomad.inbox5.com>
+
+	* oct-parse.yy (builtin): Remove seealso reference to deprecated 
+	function dispatch.
+
 2012-12-17  Konstantinos Poulios  <logari81@googlemail.com>
 
 	* graphics.h.in (class text::properties): New properties positionmode,
 	rotationmode, horizontalalignmentmode, verticalalignmentmode.
 	* graphics.cc (axes::properties::init, axes::properties::set_defaults):
 	Set mode to auto for all initialized positioning properties.
 	(axes::properties::set_xlabel,set_ylabel,set_zlabel,set_title):
 	Revert to auto-positioning mode and turn clipping off when title or
diff --git a/src/oct-parse.yy b/src/oct-parse.yy
--- a/src/oct-parse.yy
+++ b/src/oct-parse.yy
@@ -4236,18 +4236,17 @@ by name, and use @code{feval} to call th
 
   return retval;
 }
 
 DEFUN (builtin, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn {Loadable Function} {[@dots{}]} builtin (@var{f}, @dots{})\n\
 Call the base function @var{f} even if @var{f} is overloaded to\n\
-some other function for the given type signature.\n\
-@seealso{dispatch}\n\
+another function for the given type signature.\n\
 @end deftypefn")
 {
   octave_value_list retval; 
 
   int nargin = args.length ();
 
   if (nargin > 0)
     {
